server.js

```diff
@@ -33,6 +33,20 @@ var bycountry_query = `SELECT country_code, sum(total)::numeric AS total, sum(do
                        GROUP BY country_code
                        ) AS bycountry GROUP BY bycountry.country_code ORDER BY bycountry.country_code`;
 
+
+var byday_query = `SELECT day, SUM(total) AS total FROM
+                  (SELECT SUM(settle_amount)::numeric as total, DATE(timestamp) as day FROM paypal
```

Comment:
I'd change the `DATE(timestamp)` function to `date_trunc('day', timestamp)` in the various parts of this code